litmus-portal/frontend/README.md

```diff
@@ -75,115 +50,68 @@ frontend
 ├── 📃tsconfig.json
 ├── 📃tsconfig.prod.json
 ├── 📃tsconfig.test.json
-```
 
-<br>
-
-### `2. Available Scripts`
-
-<br>
+2. Available Scripts
```

Comment:
This is supposed to be a bold list item which is linked with the Table of Contents.